learning_wavelets/exact_recon_unet_evaluate.py

```diff
@@ -0,0 +1,128 @@
+import os
```

Comment:
Let's try this organization:
- in `learning_wavelets`, have an `evaluate` folder, with the following files: `__init__.py` with nothing in it, `utils.py` with the functions and class defined here (to share between the different evaluation scripts basically), `exact_recon_unet_evaluate.py` with just the function to evaluate